config/config.yaml

```diff
@@ -140,6 +140,87 @@ clouds:
       clusterService:
         imageTag: 6157c57
         imageRepo: app-sre/uhc-clusters-service
+        azureOperatorsManagedIdentitiesConfig: |
+          controlPlaneOperatorsIdentities:
+            cloud-controller-manager:
+              minOpenShiftVersion: 4.17
```

Comment:
Will this differ from Env to Env? Or will this see progressive updates? Just asking, cause this large key might make the config harder to read then (especially when thinking about later rollouts with more envs).

Reply:
It will differ from env to env and it may see updates if CS needs to add an identity..  I don't like it but I was having a hard time templatizing this object and making it work nice with config.mk.  The problem I hit was that templatize was not creating a valid multi line string in the config.mk.  To work around this I treat it as a string and base64 encode it so that it becomes a single line in config.mk and decode it in the helm template.